app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  def show
+    @user = params[:user_id]
+    @post = params[:post_id]
+  end
+
+  def index
+    @user = params[:user_id]
```

Comment:
Has users_controller and posts_controller with #index and #show actions 🐐 Well done!

Reply:
Has corresponding view files for each of the controller actions ✔